Imagine/Cache/Resolver/WebPathResolver.php

```diff
@@ -63,7 +63,7 @@ public function resolve($path, $filter)
      */
     public function isStored($path, $filter)
     {
-        return $this->filesystem->exists($this->getFilePath($path, $filter));
+        return is_file($this->getFilePath($path, $filter));
```

Comment:
(Hopping in here from Twitter! Hi Lukas!)

Wouldn't you want to call it as \is_file() instead (same for the sys_get_temp_dir() call in WebPathResolverTest.php) so that the call shaves off the resolve time instead of looking for:

> \Liip\ImagineBundle\Imagine\Cache\Resolver\is_file()
> \Liip\ImagineBundle\Imagine\Cache\is_file()
> \Liip\ImagineBundle\Imagine\is_file()
> \Liip\ImagineBundle\is_file()
> \Liip\is_file()
> \is_file()

Although I can't remember if the lookup attempts between \Liip\ImagineBundle\Imagine\Cache\Resolver\is_file() and \is_file() are still the case.